contrib/completion/bash/docker

```diff
@@ -989,6 +989,11 @@ _docker_ps() {
 			COMPREPLY=( $( compgen -W "exited paused restarting running" -- "${cur#=}" ) )
 			return
 			;;
+		*image=*)
```

Comment:
Should `image` be added to the list of completions for `--filter` above on line 968 too?